lib/matplotlib/backends/backend_pgf.py

```diff
@@ -926,21 +910,13 @@ def _print_png_to_fh(self, fh, *args, **kwargs):
             except:
                 TmpDirCleaner.add(tmpdir)
 
-    def print_png(self, fname_or_fh, *args, **kwargs):
-        """
-        Use LaTeX to compile a pgf figure to pdf and convert it to png.
-        """
+    def print_png(self, fname_or_fh, *args, dryrun=False, **kwargs):
+        """Use LaTeX to compile a pgf figure to pdf and convert it to png."""
         if kwargs.get("dryrun", False):
```

Comment:
This is now wrong since `dryrun` is now a local variable rather than stored within `kwargs`

Reply:
Yup, sorry, I realized this halfway through the patch and reverted the change, but apparently forgot to do it everywhere.